go.mod

```diff
@@ -4,17 +4,17 @@ go 1.13
 
 require (
 	github.com/RoaringBitmap/roaring v0.4.23
-	github.com/blevesearch/bleve_index_api v0.0.8
+	github.com/blevesearch/bleve_index_api v1.0.0
 	github.com/blevesearch/go-porterstemmer v1.0.3
-	github.com/blevesearch/scorch_segment_api v0.0.10
+	github.com/blevesearch/scorch_segment_api v1.0.0
 	github.com/blevesearch/segment v0.9.0
 	github.com/blevesearch/snowballstem v0.9.0
 	github.com/blevesearch/upsidedown_store_api v0.0.1
```

Comment:
The README in https://github.com/blevesearch/upsidedown_store_api needs to be updated, now that it'll be officially used. Think that needs to be done before this change here?

Reply:
oh, well ya that should have been before i tagged v1.0.0.  so much for the checklist...

Reply:
Heh, just spotted it.